fusillade-api.yml

```diff
@@ -772,7 +873,25 @@ components:
       schema:
         type: string
         enum: ['enabled', 'disabled']
+    next_token:
+      name: next_token
+      in: query
+      required: false
+      schema:
+        type: string
+    per_page:
+      name: per_page
+      in: query
+      required: false
+      schema:
+        type: integer
+        minimum: 5
+        maximum: 30
```

Comment:
I'd set this higher if possible, unless >100 times out.

Also, a default should be set: https://github.com/HumanCellAtlas/data-store/blob/master/dss-api.yml#L240

Reply:
30 is the AWS limit. In the future I can implement an index that a user could search.

Reply:
I'll add a default